src/wgpu/wgpu-surface.cpp

```
@@ -167,39 +167,40 @@ Result SurfaceImpl::init(DeviceImpl* device, WindowHandle windowHandle)
+    if (resolvedConfig.usage == TextureUsage::None)
     {
-        m_config.format = m_info.preferredFormat;
-    }
-    if (m_config.usage == TextureUsage::None)
-    {
-        m_config.usage = m_info.supportedUsage;
+        resolvedConfig.usage = (TextureUsage::Present | TextureUsage::RenderTarget | TextureUsage::CopyDestination) &
+                               m_info.supportedUsage;
     }
 
     // sRGB formats cannot be used as storage textures.
```

Cppcheck (2.21.0)

[style] The function 'rend' is never used.

(unusedFunction)